app/services/service_provider_logo_updater.rb

```diff
@@ -105,6 +105,8 @@ def dashboard_sp_url
     else
       'http://localhost:3001/api/service_providers'
     end
+  rescue LoginGov::Hostdata::MissingConfigError
```

Comment:
Looking at the source of the gem....why is this rescue needed? the `in_datacenter?` guard should cover this? Are we stubbing part of the gem or something in specs? If so, I'd look into how we can maybe not stub it, or maybe use `Hostdata.reset!` to clean up any garbage state?

Reply:
yup, too much stubbing